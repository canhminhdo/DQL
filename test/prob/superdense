#!/bin/sh

DIR=../../test/prob
cd ../../examples/prob && maude < superdense.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/superdense.out
diff $DIR/superdense.expected $DIR/superdense.out
rm $DIR/superdense.out