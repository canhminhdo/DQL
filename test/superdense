#!/bin/sh

DIR=../test
cd ../examples && maude < superdense.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/superdense.out
diff $DIR/superdense.expected $DIR/superdense.out
rm $DIR/superdense.out