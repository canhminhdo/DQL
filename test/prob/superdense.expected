==========================================
reduce in SUPERDENSE : initQState |= ((([encode10 ; decode](P(0,|1>) with prob
    == 1) /\ (P(1,|0>) with prob == 1)) /\ [encode11 ; decode](P(0,|1>) with
    prob == 1) /\ (P(1,|1>) with prob == 1)) /\ [encode01 ; decode](P(0,|0>)
    with prob == 1) /\ (P(1,|1>) with prob == 1)) /\ [encode00 ; decode](P(0,
    |0>) with prob == 1) /\ (P(1,|0>) with prob == 1) .
result JudgmentSet: emptyJS
Bye.
